test/htcmock/100_pods/redis/test_scripts/results.json

```diff
@@ -0,0 +1,126 @@
+[
```

Comment:
Why did you push the results file in the repo ?

Reply:
as an example in this PR, i will delete them later.